hphp/runtime/vm/jit/vasm-lower.cpp

```diff
@@ -189,7 +189,16 @@ void lower_vcall(Vunit& unit, Inst& inst, Vlabel b, size_t i) {
       static_assert(offsetof(TypedValue, m_type) == 8, "");
 
       if (dests.size() == 2) {
-        v << copy2{rret(0), rret(1), dests[0], dests[1]};
+        if (!(arch() == Arch::ARM)) {
```

Comment:
Could you make these switch on architecture instead?  I doubt we have plans to add any more soon, but if we ever do, it would be good to force auditing of this site.